.github/workflows/ci.yml

```diff
@@ -11,6 +11,9 @@ jobs:
       run:
         shell: bash
     steps:
+      - name: Config git
+        run: "git config --global --add safe.directory /__w/robocup-software/robocup-software"
+
```

Comment:
Due to some recent issue with git the repo being checked out was considered unsafe. Adding this before the checkout step fixes the problem.

Reply:
amazing